File: w01/d03/Tim_Hannes/air_conditioning.rb
puts "Hello, what's the current temperature?"
current_temp = gets.chomp.to_i

puts "Is the A/C functional? (y/n)"
functional = gets.chomp.downcase[0]

puts "What temperature would you like?"
desired_temp = gets.chomp.to_i

if temp > desired_temp && functional == "y"
  puts "Turn the A/C on."
elsif current_temp > desired_temp && functional == "n"
  puts "Fix the A/C please."
elsif temp < desired_temp && functional == "n"
  puts "Fix it whenever."
else
  puts "Maybe get a heater?"
end

File: w06/d01/Tim_Hannes/afternoon/air_conditioning.js
function conditionAir(current_temp, desired_temp) {
  if (current_temp > desired_temp){
    var note = 'Turn on the A/C please.';
  }
  else if (current_temp < desired_temp){
    var note = 'Turn off the A/C please.';
  }
  else {
    var note = 'Dont change a thing!'
  }
  return note;
}
